user-edits/README.md

```diff
@@ -0,0 +1,78 @@
+# Implentation of User-edits
```

Comment:
This folder, `user-edits` is old! I think it can probably be taken out of here. It is how `configure_script_yaml.py` (originally `apply-edits.py`) and the yaml development started.